internal/opts/opts.go

```diff
@@ -222,26 +216,6 @@ func ValidateIPAddress(val string) (string, error) {
 	return "", fmt.Errorf("%s is not an ip address", val)
 }
 
-// ValidateDNSSearch validates domain for resolvconf search configuration.
```

Comment:
Why remove the code?  Please put it in a different commit in case the code is dead.